src/withNetworkConnectivity.js

```diff
@@ -59,7 +59,9 @@ const withNetworkConnectivity = (
     componentDidMount() {
       NetInfo.isConnected.addEventListener(
         'connectionChange',
-        withExtraHeadRequest ? this.checkInternet : this.handleNetInfoChange,
+        withExtraHeadRequest
+          ? this.handleNetInfoChange
+          : this.handleConnectivityChange
```

Comment:
Prettier is complaining about missing a trailing comma. Sorry, just to keep a consistent code style. Once that's fixed, I'll merge it